docs/src/pages/customization/globals/globals.md

```diff
@@ -1,38 +1,66 @@
 # Globals
 
-<p class="description">The styleOverrides key enables you to customize the appearance of all instances of a component type, while the props key enables you to change the default value(s) of a component's props.</p>
+<p class="description">With the `theme.components` key you can customize the appearance of all instances of a component; change the default value(s) of a component's props; and add custom variants to your components.</p>
 
-## CSS
-
-When the configuration variables aren't powerful enough, you can take advantage of the
-`styleOverrides` key of the `theme` to potentially change **every single style** injected by Material-UI into the DOM.
-That's a really powerful feature.
+## Default props
 
-To override lab components styles with TypeScript, check [this documentation](/components/about-the-lab/#typescript).
+You can change the default props of every Material-UI component.
+A `defaultProps` key is exposed in the theme's `components` key for this use case.
 
 ```js
 const theme = createMuiTheme({
   components: {
-    // Style sheet name ⚛️
-    MuiButton: {
-      styleOverrides: {
-        // Name of the rule
-        textPrimary: {
-          // Some CSS
-          color: 'white',
-        },
+    // Name of the component ⚛️
```

Comment:
What's the asterisk for?

Reply:
It was supposed to render as ⚛️ but it doesn't seem to be visible enough

<img width="226" alt="Capture d’écran 2020-12-19 à 00 30 12" src="https://user-images.githubusercontent.com/3165635/102672549-60800c80-4191-11eb-9287-c017fb260a4c.png">

```suggestion
    // Name of the component
```